### PR TITLE
Added 3 kernels

### PR DESCRIPTION
**Pull request by romitjain**
Added 3 kernels that I had implemented while working on the implementation of ViT in triton. All of these work on batched input.

### CODE REVIEW

File: kernel_overview.md [hunk 1]
```diff
@@ -11,4 +11,6 @@
 | 7 | Swiglu | Activation | [link](kernels/0007_Swiglu.md) |
 | 8 | Gemm Int4 | Quant, Dequant, Matmul, Gemm | [link](kernels/0008_Gemm_Int4.md) |
 | 9 | Gemm Int8 | Quant, Dequant, Matmul, Gemm | [link](kernels/0009_Gemm_Int8.md) |
-| n+1 | your contribution | its tags |  |
+| 10 | Batched Conv2D | Conv | [link](kernels/0010_Conv2D.md) |
+| 11 | Batched Addition | Add | [link](kernels/0011_Add.md) |
+| 12 | Batched Patchng | Rearranging | [link](kernels/0012_Patching.md) |
```
Comment (UmerHA) [inline kernel_overview.md:16]: Could you re-add the line `| n+1 | your contribution | its tags |  |` at the end?

File: kernels/0010_Conv2D.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Batched Conv2D
+
+**Link:** https://github.com/cmeraki/vit.triton/blob/main/vit/kernels/conv2d.py
+
+**Author:** Romit
```
Comment (UmerHA) [inline kernels/0010_Conv2D.md]: If you want, feel free to add your full name, with a link to your twitter or LinkedIn. Eg `[Romit last_name](x.com/blabla)`.

Same in other files

**Review by UmerHA — CHANGES_REQUESTED** [review]
Look's very good! I've requested / suggested some tiny changes. After that, I'll merge!

File: kernel_overview.md (hunk 1 above)
Comment (romitjain) [reply kernel_overview.md:16]: Done

**Comment by romitjain** [comment]
@UmerHA Thanks for the review. I have added your suggestions.

**Comment by UmerHA** [comment]
Thanks!

**Pull request merged**